### ISSUE TITLE
Visualizing specific layers Tensorboard Callback

### ISSUE THREAD

**Issue by rodyt** [keras-team/keras#13745]
Keras's Tensorboard callback has a write_images and a histogram_freq argument that allows the weights and activations to be saved to Tensorboard to visualize during training. This is helpful for visualizing training in real time.

The issue is: this saves the information for every layer and makes Tensorboard very messy to read, especially if I have logged other images to Tensorboard. This can be seen in the images below:

![Images2](https://user-images.githubusercontent.com/36773880/73796740-e73b1f00-4762-11ea-8c1e-5da46d118c83.png)
![Images1](https://user-images.githubusercontent.com/36773880/73796742-e7d3b580-4762-11ea-888e-558a93bcb801.png)

A lot of this logged information is redundant.

Is there any way to make the weight distribution and activation visualizations more organized so that they don't fill up my whole screen?

Is there any way to only visualize certain layers using the Keras callback? (i.e. I don't want to visualize the BatchNorm layers)

**Comment by Saduf2019** [comment]
@rodyt 
Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
